### DISCUSSION TITLE
Importing data from Google Analytics

### DISCUSSION THREAD

**Discussion by ErisFY** [plausible/analytics#958] [Anything Else]
I know for sure this one feature has already been requested, but I just wanted to make it known that newcomers to Plausible (such as myself) are also very much interested in this potential feature. I've used Google Analytics for over a year now, and have looking to transition from it, but I'm not too keen on losing all that data. 

I understand that GA also has far more features than Plausible, so understandably not everything would carry over to Plausible, but I would still appreciate seeing those few basic information (like visits counts, etc) to be imported.

**Comment by metmarkosaric** [comment]
we'd like to allow this at some point. see #211

**Comment by metmarkosaric** [comment]
Universal Analytics, the current version of Google Analytics, [will be sunset](https://plausible.io/blog/universal-google-analytics-is-dead) and will stop counting stats on July 1st 2023. There’s no way to import your historical Universal Analytics stats into the new Google Analytics 4, so we've put a lot of effort into making a Google Analytics stats import tool ourselves. 

You can now import your historical Google Analytics (Universal Analytics) stats into your Plausible dashboard. You can import the stats from your first Google Analytics visitor until your first Plausible Analytics visitor. [See more here](https://plausible.io/docs/google-analytics-import).
